README.md

```diff
@@ -3,3 +3,4 @@ Just to test some github workflow assumption
 
 ### Can I rebase a PR branch on recent master?
```

Comment:
Yep, previous line comment stood still with the text, it was commenting, moving with line number.